MAUI/Parallax-View/Getting-started.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Parallax View control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Parallax View (SfParallaxView) control, its elements and more.
```

Comment:
description: Learn about getting started with Syncfusion .NET MAUI Parallax View (SfParallaxView) control, its elements, and more.